pkg/cortex/cortex.go

```diff
@@ -0,0 +1,111 @@
+// Package cortex provides methods to query data from our central
+// Prometheus data store.
+package cortex
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/config"
+	"github.com/prometheus/common/model"
+)
+
+const (
+	query = `aggregation:giantswarm:cluster_release_version{pipeline="testing"}`
+)
+
+type Config struct {
+	// Base URL of the prometheus API server.
```

Comment:
```suggestion
	// URL of the prometheus API server.
```

These need to start with the variable name to be recognised as godoc.